retis/src/core/tracking/bpf/include/skb_tracking.h

```diff
@@ -161,13 +186,14 @@ static __always_inline int track_skb_end(struct retis_context *ctx)
 {
 	struct tracking_config *cfg;
 	u64 head, ksym = ctx->ksym;
+	struct tracking_info *ti;
 	struct sk_buff *skb;
 
 	cfg = bpf_map_lookup_elem(&tracking_config_map, &ksym);
 	if (!cfg)
 		return 0;
 
-	/* We only supports free functions below */
+	/* We only supports   functions below */
```

Comment:
Looks like the "free" removal was a bit too broad :)

(There's at least another one)